.github/workflows/ci.yml

```diff
@@ -18,12 +18,14 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: '3.9'
-      - run: python -m pip install --upgrade pip
-      - run: python -m pip install -r requirements/ci.txt
-      - run: tox -e static
-      - uses: stefanzweifel/git-auto-commit-action@v5
+      - uses: pre-commit/action@v3.0.0
```

Comment:
Not the latest version. Should we actually pin down to the patch version? We don't do this elsewhere.

Reply:
What do you mean by pin down...? 
And yeah, I see they 3.0.1 is released.

Reply:
I meant 'down to' as a preposition. I.e., pin all parts of the version number. I think we often only pin the major and maybe minor version. But not the patch version.

Reply:
Ahh like that...! Okay, then I'll get rid of `.0.0`

Reply:
Apparently this doesn't work. In that case, I'm fine with using the full version

Reply:
@jl-wynen Apparently `pre-commit` ci action doesn't work like that...

Reply:
https://github.com/scipp/copier_template/actions/runs/7841440116/job/21397772718